docs/MERGE-CHECKLIST.md

```
@@ -0,0 +1,221 @@
+# 5. Open PR on GitHub
+# Target: main
+# Use the PR template below
+```
+
+---
+
+## 3. Pull Request Requirements
+
+### PR Title Format
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)